packages/common/src/common.ts

```diff
@@ -1,7 +1,9 @@
+import { Hex } from "viem";
 import { ResourceType } from "./resourceTypes";
 
-export type ResourceId = {
+export type Resource = {
   namespace: string;
   name: string;
   type: ResourceType;
+  hex: Hex;
```

Comment:
should arguably call this `resourceId` if we're calling the object "resource"

Reply:
agree this could be `resourceId`